src/components/productFilter/Filter.jsx

```diff
@@ -0,0 +1,201 @@
+import filter from "./filter.module.css";
```

Comment:
You can create components for each feature like a component for rating, a component for price range and import, so that you code looks more clean